tests/integ/kms_utils.py

```diff
@@ -42,22 +48,72 @@ def _get_kms_key_arn(kms_client, alias):
         return None
 
 
-def _create_kms_key(kms_client, account_id):
+def _get_kms_key_id(kms_client, alias):
+    try:
+        response = kms_client.describe_key(KeyId='alias/' + alias)
+        return response['KeyMetadata']['KeyId']
+    except kms_client.exceptions.NotFoundException:
+        return None
+
+
+def _create_kms_key(kms_client,
+                    account_id,
+                    role_arn=None,
+                    sagemaker_role='SageMakerRole'):
+    if role_arn:
+        principal = PRINCIPAL_TEMPLATE.format(account_id=account_id,
```

Comment:
is the idea here a policy that allows the root account, some role arn, plus another role constructed from the sagemaker_role name? why not simplify that and just require test environments be set up correctly?

Reply:
Talked with @jesterhazy offline. We didn't find an easier way to simplify this test without breaking running tests outside the CI.